db/migrate/20210315162119_add_email_sent_at_to_education_stem_automated_decision.rb

```diff
@@ -0,0 +1,6 @@
+class AddEmailSentAtToEducationStemAutomatedDecision < ActiveRecord::Migration[6.0]
```

Comment:
This migration and the schema change will need to be removed to their own PR. Dangerbot won't allow any other changes to be included with migrations.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/6176